### PR TITLE
Mark SECURITY-1523 as fixed in sonar-quality-gates-plugin 312.vcde107…

### PR DESCRIPTION
**Pull request by kirk-fitz**
Fix was in [https://github.com/jenkinsci/sonar-quality-gates-plugin/pull/30](https://github.com/jenkinsci/sonar-quality-gates-plugin/pull/30)

### CODE REVIEW

**Comment by daniel-beck** [comment]
Noting https://groups.google.com/g/jenkinsci-dev/c/Qh8oJo1iYFk/m/kGKKB8q1AgAJ

> Looking at the UI, I would like the token to be stored and transmitted as a Secret and masked on the UI, before removing this warning, That's essentially the same issue, and I cannot reconstruct why we didn't pick up on it at the time.

This doesn't seem to be changed, without explanation.

**Comment by kirk-fitzsimons** [comment]
The changes were made here https://github.com/jenkinsci/sonar-quality-gates-plugin/pull/30/commits/60d715cb6bac35273cad8c36e9f3c740209aa9b5

**Comment by kirk-fitz** [comment]
Is my fix not the same as was applied and approved in the backlog plugin?

https://github.com/jenkins-infra/update-center2/pull/509

https://github.com/jenkinsci/backlog-plugin/commit/43f513380226831ed914c85e1bfff1cda296bbf7

**Comment by kirk-fitz** [comment]
Apologies, i was looking at https://groups.google.com/g/jenkinsci-dev/c/Qh8oJo1iYFk/m/R4vycl6jAgAJ which mentioned the password, I see later you mentioned the token, I will try fix the token

**Comment by daniel-beck** [comment]
Thanks. Link the PR and I'll take a look asap.

**Comment by kirk-fitz** [comment]
https://github.com/jenkinsci/sonar-quality-gates-plugin/pull/34

I ran the tests locally and they work, will check with the maintainer if it looks good, can you let me know if you think its good

**Comment by kirk-fitz** [comment]
https://github.com/jenkinsci/sonar-quality-gates-plugin/pull/34 has been merged

And latest release contains the fix 
https://github.com/jenkinsci/sonar-quality-gates-plugin/releases/tag/314.v0a_35c497299b_

Have updated warnings.json, I believe the regex to be correct

**Comment by kirk-fitzsimons** [comment]
@daniel-beck Can you have a look when you get a chance please

**Comment by daniel-beck** [comment]
@kirk-fitzsimons I plan to take a look later this week.

**Comment by kirk-fitz** [comment]
Great, thanks Daniel

File: resources/warnings.json
Comment (daniel-beck) [inline resources/warnings.json:7342]: All `1.*` can be collapsed; 309 was improperly removed in the update; and we match affected, not fixed, so 312 instead of 314.

```suggestion
        "lastVersion": "312.vcde1075036ef",
        "pattern": "(1|309|312)(|[.-].+)"
```

**Review by daniel-beck — APPROVED** [review]

**Comment by kirk-fitzsimons** [comment]
Thanks Daniel, sorry i misunderstood the regex, thank you for fixing, once it is merged how long does it take to get updated?

**Comment by daniel-beck** [comment]
@kirk-fitzsimons Update sites have it within 10 minutes (but Jenkins caches this data for up to 24 hrs), plugins.jenkins.io in a few hours or so.

**Pull request merged**
